src/api/lib/backend/doc/Backend.html

```diff
@@ -6,7 +6,7 @@
 <title>
   Module: Backend
   
-    &mdash; Documentation by YARD 0.9.9
+    &mdash; Documentation by YARD 0.9.12
```

Comment:
What are these files and what do they have to do with unifying the json encoders? :thinking:

Reply:
Just updated the documentation because we changed one method in the Backend::Api lib...

Reply:
Ok, I wasn't aware that we had this kind of documentation, I guess I missed something..